### ISSUE TITLE
Soul Walk and Vampiric Curse (2 easy power tweaks to help necros)

### ISSUE THREAD

**Issue by GeneralGormal** [torilmud/issues#885]
Necromancers need some love in general, but since Fugue was added and soul walk changed, it's a 9th circle quest spell that does the same thing as raise dead, the freebie that all priests get.  It needs to be 50% fatigue to match res.

Vamp Curse does not land often enough for a single target, relatively minor vamp heal to make it worth casting except in a handful of situations.  Make the spell save less punishing and buff the healing slightly or just remove the save entirely.

The class needs a larger overhaul, but as they are right now, there's no reason to not lich.  Even when Soul Walk was desired, you only ever saw a necro show up for Tiamat corpse detail to save the group time ressing during the fights.  Now it costs you time to soul walk over res.  Vamp Curse has never been good.

**Comment by shevarash** [comment]
You're right that this class has much larger issues. I don't think that tweaking Vamp Curse or Soul Walk will make much difference, but they're also pretty easy tweaks so I'm happy to make them while Necros wait for a larger update.
